packages/checkout/sdk/src/types/config.ts

```diff
@@ -125,6 +125,8 @@ export type DexConfig = {
   overrides?: ExchangeOverrides;
   /** An array of tokens compatible with the DEX. */
   tokens?: TokenInfo[];
+  /** An array of secondary fees to be applied to swaps */
+  secondaryFees?: SecondaryFee[];
```

Comment:
The config will include `secondaryFees` when this PR is merged https://github.com/immutable/checkout-api-configs/pull/50/files.

In terms of backwards compatibility, because `secondaryFees` are optional, versions prior to this will still be able fetch the new config without any issues. LMK if you think this could cause issues